std_msgs/Header header
gvgexploration/Ridge[] edges
gvgexploration/Pixel[] pixels